File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Product Showcase/VS Demo/VB/VSDemoViewItems/ClassView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Windows.Forms

Namespace VSDemo
	''' <summary>
	''' Summary description for ClassView.
	''' </summary>
	Public Class ClassView : Inherits System.Windows.Forms.UserControl
		Private panelClassView As System.Windows.Forms.Panel
		Public classViewToolBar As Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar
		Private sortBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.BarItem
		Private newFolderBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.BarItem
		Private dockingImageList As System.Windows.Forms.ImageList
		Private tvClassView As System.Windows.Forms.TreeView
		Private components As System.ComponentModel.IContainer

		Public Sub New()
			' This call is required by the Windows.Forms Form Designer.
			InitializeComponent()

			' TODO: Add any initialization after the InitForm call

		End Sub

		''' <summary> 
		''' Clean up any resources being used.
		''' </summary>
		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		#Region "Component Designer generated code"
		''' <summary> 
		''' Required method for Designer support - do not modify 
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.components = New System.ComponentModel.Container()
			Dim treeNode1 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("MainForm()", 72, 72)
			Dim treeNode2 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("Dispose(bool)", 70, 71)
			Dim treeNode3 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("InitializeComponent()", 70, 70)
			Dim treeNode4 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("components", 73, 73)
			Dim treeNode5 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("Program", 69, 69, New System.Windows.Forms.TreeNode() { treeNode1, treeNode2, treeNode3, treeNode4})
			Dim treeNode6 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("ConsoleApplication", 68, 68, New System.Windows.Forms.TreeNode() { treeNode5})
			Dim treeNode7 As System.Windows.Forms.TreeNode = New System.Windows.Forms.TreeNode("ConsoleApplication1", 61, 61, New System.Windows.Forms.TreeNode() { treeNode6})
			Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(ClassView))
			Me.panelClassView = New System.Windows.Forms.Panel()
			Me.tvClassView = New System.Windows.Forms.TreeView()
			Me.dockingImageList = New System.Windows.Forms.ImageList(Me.components)
			Me.classViewToolBar = New Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar()
			Me.sortBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem()
			Me.newFolderBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem()
			Me.panelClassView.SuspendLayout()
			Me.SuspendLayout()
			' 
			' panelClassView
			' 
			Me.panelClassView.Controls.Add(Me.tvClassView)
			Me.panelClassView.Controls.Add(Me.classViewToolBar)
			Me.panelClassView.Dock = System.Windows.Forms.DockStyle.Fill
			Me.panelClassView.Location = New System.Drawing.Point(0, 0)
			Me.panelClassView.Name = "panelClassView"
			Me.panelClassView.Size = New System.Drawing.Size(184, 568)
			Me.panelClassView.TabIndex = 0
			' 
			' tvClassView
			' 
			Me.tvClassView.Dock = System.Windows.Forms.DockStyle.Fill
			Me.tvClassView.Font = New System.Drawing.Font("Calibri", 9.75F)
			Me.tvClassView.ImageIndex = 0
			Me.tvClassView.ImageList = Me.dockingImageList
			Me.tvClassView.Location = New System.Drawing.Point(0, 25)
			Me.tvClassView.Name = "tvClassView"
			treeNode1.ImageIndex = 72
			treeNode1.Name = ""
			treeNode1.SelectedImageIndex = 72
			treeNode1.Text = "MainForm()"
			treeNode2.ImageIndex = 70
			treeNode2.Name = ""
			treeNode2.SelectedImageIndex = 71
			treeNode2.Text = "Dispose(bool)"
			treeNode3.ImageIndex = 70
			treeNode3.Name = ""
			treeNode3.SelectedImageIndex = 70
			treeNode3.Text = "InitializeComponent()"
			treeNode4.ImageIndex = 73
			treeNode4.Name = ""
			treeNode4.SelectedImageIndex = 73
			treeNode4.Text = "components"
			treeNode5.ImageIndex = 69
			treeNode5.Name = ""
			treeNode5.SelectedImageIndex = 69
			treeNode5.Text = "Program"
			treeNode6.ImageIndex = 68
			treeNode6.Name = ""
			treeNode6.SelectedImageIndex = 68
			treeNode6.Text = "ConsoleApplication"
			treeNode7.ImageIndex = 61
			treeNode7.Name = ""
			treeNode7.SelectedImageIndex = 61
			treeNode7.Text = "ConsoleApplication1"
			Me.tvClassView.Nodes.AddRange(New System.Windows.Forms.TreeNode() { treeNode7})
			Me.tvClassView.SelectedImageIndex = 0
			Me.tvClassView.Size = New System.Drawing.Size(184, 543)
			Me.tvClassView.TabIndex = 30
			' 
			' dockingImageList
			' 
			Me.dockingImageList.ImageStream = (CType(resources.GetObject("dockingImageList.ImageStream"), System.Windows.Forms.ImageListStreamer))
			Me.dockingImageList.TransparentColor = System.Drawing.Color.Transparent
			Me.dockingImageList.Images.SetKeyName(0, "")
			Me.dockingImageList.Images.SetKeyName(1, "")
			Me.dockingImageList.Images.SetKeyName(2, "")
			Me.dockingImageList.Images.SetKeyName(3, "")
			Me.dockingImageList.Images.SetKeyName(4, "")
			Me.dockingImageList.Images.SetKeyName(5, "")
			Me.dockingImageList.Images.SetKeyName(6, "")
			Me.dockingImageList.Images.SetKeyName(7, "")
			Me.dockingImageList.Images.SetKeyName(8, "")
			Me.dockingImageList.Images.SetKeyName(9, "")
			Me.dockingImageList.Images.SetKeyName(10, "")
			Me.dockingImageList.Images.SetKeyName(11, "")
			Me.dockingImageList.Images.SetKeyName(12, "")
			Me.dockingImageList.Images.SetKeyName(13, "")
			Me.dockingImageList.Images.SetKeyName(14, "")
			Me.dockingImageList.Images.SetKeyName(15, "")
			Me.dockingImageList.Images.SetKeyName(16, "")
			Me.dockingImageList.Images.SetKeyName(17, "")
			Me.dockingImageList.Images.SetKeyName(18, "")
			Me.dockingImageList.Images.SetKeyName(19, "")
			Me.dockingImageList.Images.SetKeyName(20, "")
			Me.dockingImageList.Images.SetKeyName(21, "")
			Me.dockingImageList.Images.SetKeyName(22, "")
			Me.dockingImageList.Images.SetKeyName(23, "")
			Me.dockingImageList.Images.SetKeyName(24, "")
			Me.dockingImageList.Images.SetKeyName(25, "")
			Me.dockingImageList.Images.SetKeyName(26, "")
			Me.dockingImageList.Images.SetKeyName(27, "")
			Me.dockingImageList.Images.SetKeyName(28, "")
			Me.dockingImageList.Images.SetKeyName(29, "")
			Me.dockingImageList.Images.SetKeyName(30, "")
			Me.dockingImageList.Images.SetKeyName(31, "")
			Me.dockingImageList.Images.SetKeyName(32, "")
			Me.dockingImageList.Images.SetKeyName(33, "")
			Me.dockingImageList.Images.SetKeyName(34, "")
			Me.dockingImageList.Images.SetKeyName(35, "")
			Me.dockingImageList.Images.SetKeyName(36, "")
			Me.dockingImageList.Images.SetKeyName(37, "")
			Me.dockingImageList.Images.SetKeyName(38, "")
			Me.dockingImageList.Images.SetKeyName(39, "")
			Me.dockingImageList.Images.SetKeyName(40, "")
			Me.dockingImageList.Images.SetKeyName(41, "")
			Me.dockingImageList.Images.SetKeyName(42, "")
			Me.dockingImageList.Images.SetKeyName(43, "")
			Me.dockingImageList.Images.SetKeyName(44, "")
			Me.dockingImageList.Images.SetKeyName(45, "")
			Me.dockingImageList.Images.SetKeyName(46, "")
			Me.dockingImageList.Images.SetKeyName(47, "")
			Me.dockingImageList.Images.SetKeyName(48, "")
			Me.dockingImageList.Images.SetKeyName(49, "")
			Me.dockingImageList.Images.SetKeyName(50, "")
			Me.dockingImageList.Images.SetKeyName(51, "")
			Me.dockingImageList.Images.SetKeyName(52, "")
			Me.dockingImageList.Images.SetKeyName(53, "")
			Me.dockingImageList.Images.SetKeyName(54, "")
			Me.dockingImageList.Images.SetKeyName(55, "")
			Me.dockingImageList.Images.SetKeyName(56, "")
			Me.dockingImageList.Images.SetKeyName(57, "")
			Me.dockingImageList.Images.SetKeyName(58, "")
			Me.dockingImageList.Images.SetKeyName(59, "")
			Me.dockingImageList.Images.SetKeyName(60, "")
			Me.dockingImageList.Images.SetKeyName(61, "")
			Me.dockingImageList.Images.SetKeyName(62, "")
			Me.dockingImageList.Images.SetKeyName(63, "")
			Me.dockingImageList.Images.SetKeyName(64, "")
			Me.dockingImageList.Images.SetKeyName(65, "")
			Me.dockingImageList.Images.SetKeyName(66, "")
			Me.dockingImageList.Images.SetKeyName(67, "")
			Me.dockingImageList.Images.SetKeyName(68, "")
			Me.dockingImageList.Images.SetKeyName(69, "")
			Me.dockingImageList.Images.SetKeyName(70, "")
			Me.dockingImageList.Images.SetKeyName(71, "")
			Me.dockingImageList.Images.SetKeyName(72, "")
			Me.dockingImageList.Images.SetKeyName(73, "")
			Me.dockingImageList.Images.SetKeyName(74, "")
			Me.dockingImageList.Images.SetKeyName(75, "")
			Me.dockingImageList.Images.SetKeyName(76, "")
			Me.dockingImageList.Images.SetKeyName(77, "")
			Me.dockingImageList.Images.SetKeyName(78, "")
			' 
			' classViewToolBar
			' 
			Me.classViewToolBar.BackColor = System.Drawing.Color.Transparent
			Me.classViewToolBar.Dock = System.Windows.Forms.DockStyle.Top
			Me.classViewToolBar.Items.AddRange(New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem() { Me.sortBarItem, Me.newFolderBarItem})
			Me.classViewToolBar.Location = New System.Drawing.Point(0, 0)
			Me.classViewToolBar.Name = "classViewToolBar"
			Me.classViewToolBar.Size = New System.Drawing.Size(184, 25)
			Me.classViewToolBar.Style = Syncfusion.Windows.Forms.VisualStyle.Office2007
			Me.classViewToolBar.TabIndex = 0
			' 
			' sortBarItem
			' 
			Me.sortBarItem.ImageIndex = 36
			Me.sortBarItem.ImageList = Me.dockingImageList
			Me.sortBarItem.UpdateUIOnAppIdle = True
			' 
			' newFolderBarItem
			' 
			Me.newFolderBarItem.ImageIndex = 37
			Me.newFolderBarItem.ImageList = Me.dockingImageList
			' 
			' ClassView
			' 
			Me.Controls.Add(Me.panelClassView)
			Me.Name = "ClassView"
			Me.Size = New System.Drawing.Size(184, 568)
			Me.panelClassView.ResumeLayout(False)
			Me.ResumeLayout(False)

		End Sub
		#End Region
	End Class
End Namespace

File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Product Showcase/VS Demo/CS/VSDemoViewItems/ClassView.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Windows.Forms;

namespace VSDemo
{
	/// <summary>
	/// Summary description for ClassView.
	/// </summary>
	public class ClassView : System.Windows.Forms.UserControl
	{
		private System.Windows.Forms.Panel panelClassView;
		public Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar classViewToolBar;
		private Syncfusion.Windows.Forms.Tools.XPMenus.BarItem sortBarItem;
		private Syncfusion.Windows.Forms.Tools.XPMenus.BarItem newFolderBarItem;
		private System.Windows.Forms.ImageList dockingImageList;
		private System.Windows.Forms.TreeView tvClassView;
		private System.ComponentModel.IContainer components;

		public ClassView()
		{
			// This call is required by the Windows.Forms Form Designer.
			InitializeComponent();

			// TODO: Add any initialization after the InitForm call

		}

		/// <summary> 
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Component Designer generated code
		/// <summary> 
		/// Required method for Designer support - do not modify 
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
            this.components = new System.ComponentModel.Container();
            System.Windows.Forms.TreeNode treeNode1 = new System.Windows.Forms.TreeNode("MainForm()", 72, 72);
            System.Windows.Forms.TreeNode treeNode2 = new System.Windows.Forms.TreeNode("Dispose(bool)", 70, 71);
            System.Windows.Forms.TreeNode treeNode3 = new System.Windows.Forms.TreeNode("InitializeComponent()", 70, 70);
            System.Windows.Forms.TreeNode treeNode4 = new System.Windows.Forms.TreeNode("components", 73, 73);
            System.Windows.Forms.TreeNode treeNode5 = new System.Windows.Forms.TreeNode("Program", 69, 69, new System.Windows.Forms.TreeNode[] {
            treeNode1,
            treeNode2,
            treeNode3,
            treeNode4});
            System.Windows.Forms.TreeNode treeNode6 = new System.Windows.Forms.TreeNode("ConsoleApplication", 68, 68, new System.Windows.Forms.TreeNode[] {
            treeNode5});
            System.Windows.Forms.TreeNode treeNode7 = new System.Windows.Forms.TreeNode("ConsoleApplication1", 61, 61, new System.Windows.Forms.TreeNode[] {
            treeNode6});
            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(ClassView));
            this.panelClassView = new System.Windows.Forms.Panel();
            this.tvClassView = new System.Windows.Forms.TreeView();
            this.dockingImageList = new System.Windows.Forms.ImageList(this.components);
            this.classViewToolBar = new Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar();
            this.sortBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem();
            this.newFolderBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem();
            this.panelClassView.SuspendLayout();
            this.SuspendLayout();
            // 
            // panelClassView
            // 
            this.panelClassView.Controls.Add(this.tvClassView);
            this.panelClassView.Controls.Add(this.classViewToolBar);
            this.panelClassView.Dock = System.Windows.Forms.DockStyle.Fill;
            this.panelClassView.Location = new System.Drawing.Point(0, 0);
            this.panelClassView.Name = "panelClassView";
            this.panelClassView.Size = new System.Drawing.Size(184, 568);
            this.panelClassView.TabIndex = 0;
            // 
            // tvClassView
            // 
            this.tvClassView.Dock = System.Windows.Forms.DockStyle.Fill;
            this.tvClassView.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F);
            this.tvClassView.ImageIndex = 0;
            this.tvClassView.ImageList = this.dockingImageList;
            this.tvClassView.Location = new System.Drawing.Point(0, 25);
            this.tvClassView.Name = "tvClassView";
            treeNode1.ImageIndex = 72;
            treeNode1.Name = "";
            treeNode1.SelectedImageIndex = 72;
            treeNode1.Text = "MainForm()";
            treeNode2.ImageIndex = 70;
            treeNode2.Name = "";
            treeNode2.SelectedImageIndex = 71;
            treeNode2.Text = "Dispose(bool)";
            treeNode3.ImageIndex = 70;
            treeNode3.Name = "";
            treeNode3.SelectedImageIndex = 70;
            treeNode3.Text = "InitializeComponent()";
            treeNode4.ImageIndex = 73;
            treeNode4.Name = "";
            treeNode4.SelectedImageIndex = 73;
            treeNode4.Text = "components";
            treeNode5.ImageIndex = 69;
            treeNode5.Name = "";
            treeNode5.SelectedImageIndex = 69;
            treeNode5.Text = "Program";
            treeNode6.ImageIndex = 68;
            treeNode6.Name = "";
            treeNode6.SelectedImageIndex = 68;
            treeNode6.Text = "ConsoleApplication";
            treeNode7.ImageIndex = 61;
            treeNode7.Name = "";
            treeNode7.SelectedImageIndex = 61;
            treeNode7.Text = "ConsoleApplication1";
            this.tvClassView.Nodes.AddRange(new System.Windows.Forms.TreeNode[] {
            treeNode7});
            this.tvClassView.SelectedImageIndex = 0;
            this.tvClassView.Size = new System.Drawing.Size(184, 543);
            this.tvClassView.TabIndex = 30;
            // 
            // dockingImageList
            // 
            this.dockingImageList.ImageStream = ((System.Windows.Forms.ImageListStreamer)(resources.GetObject("dockingImageList.ImageStream")));
            this.dockingImageList.TransparentColor = System.Drawing.Color.Transparent;
            this.dockingImageList.Images.SetKeyName(0, "");
            this.dockingImageList.Images.SetKeyName(1, "");
            this.dockingImageList.Images.SetKeyName(2, "");
            this.dockingImageList.Images.SetKeyName(3, "");
            this.dockingImageList.Images.SetKeyName(4, "");
            this.dockingImageList.Images.SetKeyName(5, "");
            this.dockingImageList.Images.SetKeyName(6, "");
            this.dockingImageList.Images.SetKeyName(7, "");
            this.dockingImageList.Images.SetKeyName(8, "");
            this.dockingImageList.Images.SetKeyName(9, "");
            this.dockingImageList.Images.SetKeyName(10, "");
            this.dockingImageList.Images.SetKeyName(11, "");
            this.dockingImageList.Images.SetKeyName(12, "");
            this.dockingImageList.Images.SetKeyName(13, "");
            this.dockingImageList.Images.SetKeyName(14, "");
            this.dockingImageList.Images.SetKeyName(15, "");
            this.dockingImageList.Images.SetKeyName(16, "");
            this.dockingImageList.Images.SetKeyName(17, "");
            this.dockingImageList.Images.SetKeyName(18, "");
            this.dockingImageList.Images.SetKeyName(19, "");
            this.dockingImageList.Images.SetKeyName(20, "");
            this.dockingImageList.Images.SetKeyName(21, "");
            this.dockingImageList.Images.SetKeyName(22, "");
            this.dockingImageList.Images.SetKeyName(23, "");
            this.dockingImageList.Images.SetKeyName(24, "");
            this.dockingImageList.Images.SetKeyName(25, "");
            this.dockingImageList.Images.SetKeyName(26, "");
            this.dockingImageList.Images.SetKeyName(27, "");
            this.dockingImageList.Images.SetKeyName(28, "");
            this.dockingImageList.Images.SetKeyName(29, "");
            this.dockingImageList.Images.SetKeyName(30, "");
            this.dockingImageList.Images.SetKeyName(31, "");
            this.dockingImageList.Images.SetKeyName(32, "");
            this.dockingImageList.Images.SetKeyName(33, "");
            this.dockingImageList.Images.SetKeyName(34, "");
            this.dockingImageList.Images.SetKeyName(35, "");
            this.dockingImageList.Images.SetKeyName(36, "");
            this.dockingImageList.Images.SetKeyName(37, "");
            this.dockingImageList.Images.SetKeyName(38, "");
            this.dockingImageList.Images.SetKeyName(39, "");
            this.dockingImageList.Images.SetKeyName(40, "");
            this.dockingImageList.Images.SetKeyName(41, "");
            this.dockingImageList.Images.SetKeyName(42, "");
            this.dockingImageList.Images.SetKeyName(43, "");
            this.dockingImageList.Images.SetKeyName(44, "");
            this.dockingImageList.Images.SetKeyName(45, "");
            this.dockingImageList.Images.SetKeyName(46, "");
            this.dockingImageList.Images.SetKeyName(47, "");
            this.dockingImageList.Images.SetKeyName(48, "");
            this.dockingImageList.Images.SetKeyName(49, "");
            this.dockingImageList.Images.SetKeyName(50, "");
            this.dockingImageList.Images.SetKeyName(51, "");
            this.dockingImageList.Images.SetKeyName(52, "");
            this.dockingImageList.Images.SetKeyName(53, "");
            this.dockingImageList.Images.SetKeyName(54, "");
            this.dockingImageList.Images.SetKeyName(55, "");
            this.dockingImageList.Images.SetKeyName(56, "");
            this.dockingImageList.Images.SetKeyName(57, "");
            this.dockingImageList.Images.SetKeyName(58, "");
            this.dockingImageList.Images.SetKeyName(59, "");
            this.dockingImageList.Images.SetKeyName(60, "");
            this.dockingImageList.Images.SetKeyName(61, "");
            this.dockingImageList.Images.SetKeyName(62, "");
            this.dockingImageList.Images.SetKeyName(63, "");
            this.dockingImageList.Images.SetKeyName(64, "");
            this.dockingImageList.Images.SetKeyName(65, "");
            this.dockingImageList.Images.SetKeyName(66, "");
            this.dockingImageList.Images.SetKeyName(67, "");
            this.dockingImageList.Images.SetKeyName(68, "");
            this.dockingImageList.Images.SetKeyName(69, "");
            this.dockingImageList.Images.SetKeyName(70, "");
            this.dockingImageList.Images.SetKeyName(71, "");
            this.dockingImageList.Images.SetKeyName(72, "");
            this.dockingImageList.Images.SetKeyName(73, "");
            this.dockingImageList.Images.SetKeyName(74, "");
            this.dockingImageList.Images.SetKeyName(75, "");
            this.dockingImageList.Images.SetKeyName(76, "");
            this.dockingImageList.Images.SetKeyName(77, "");
            this.dockingImageList.Images.SetKeyName(78, "");
            // 
            // classViewToolBar
            // 
            this.classViewToolBar.BackColor = System.Drawing.Color.Transparent;
            this.classViewToolBar.Dock = System.Windows.Forms.DockStyle.Top;
            this.classViewToolBar.Items.AddRange(new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem[] {
            this.sortBarItem,
            this.newFolderBarItem});
            this.classViewToolBar.Location = new System.Drawing.Point(0, 0);
            this.classViewToolBar.Name = "classViewToolBar";
            this.classViewToolBar.Size = new System.Drawing.Size(184, 25);
            this.classViewToolBar.Style = Syncfusion.Windows.Forms.VisualStyle.Office2007;
            this.classViewToolBar.TabIndex = 0;
            // 
            // sortBarItem
            // 
            this.sortBarItem.ImageIndex = 36;
            this.sortBarItem.ImageList = this.dockingImageList;
            this.sortBarItem.UpdateUIOnAppIdle = true;
            // 
            // newFolderBarItem
            // 
            this.newFolderBarItem.ImageIndex = 37;
            this.newFolderBarItem.ImageList = this.dockingImageList;
            // 
            // ClassView
            // 
            this.Controls.Add(this.panelClassView);
            this.Name = "ClassView";
            this.Size = new System.Drawing.Size(184, 568);
            this.panelClassView.ResumeLayout(false);
            this.ResumeLayout(false);

		}
		#endregion
	}
}
